apps/settings/js/panels/operator_settings/models/operator_manager.js

```diff
@@ -142,21 +142,22 @@ define(function(require) {
 
     this.debug('search');
     this._searching = true;
-    return this._connWrapper.search().then((networks = []) => {
-      var operators = networks.map((network) => OperatorItem(network));
-      operators.forEach((operator) => {
+    return this._connWrapper.search().then(function(networks = []) {
+      var operators = networks.map(function(network) {
+        return OperatorItem(network); });
+      operators.forEach(function(operator) {
         if (operator.network.state === OperatorItem.STATE.CONNECTED) {
           this._connectedOperatorItem = operator;
         }
       });
       this._operators.reset(operators);
-    }).catch((error) => {
+    }).catch(function(error) {
```

Comment:
please keep the origin syntax(preffered), or you need add `.bind(this)` at the end of the function.